tests/test_other.py

```diff
@@ -988,6 +988,30 @@ def test_link_group_bitcode(self):
     self.run_process([EMCC, 'out.o'])
     self.assertContained('Hello', self.run_js('a.out.js'))
 
+  def test_link_single_ar_bitcode(self):
```

Comment:
Can you put this test alongside the existing `test_combining_object_files` ?

Reply:
Done, renamed to `test_combining_object_files_into_archive`.